### PR TITLE
Package callbacks cleaning

### PR DESCRIPTION
**Pull request by jreidinger**
recommended to review per commit.
still WIP, do not merge yet ( it is here for early comments )
you can see improvement in metrics -

### CODE REVIEW

**Comment by lslezak** [comment]
Syntax check reports several errors like `library/packages/src/modules/PackageCallbacks.rb:427: syntax error, unexpected ')', expecting keyword_end`

File: library/packages/src/modules/PackageCallbacks.rb [hunk 1]
```diff
@@ -3762,6 +3612,110 @@ def PackageCallbacks
...
+          ReplacePoint(Id(:info), Empty()),
+          HSpacing(0.1)
+        ),
+        HBox(HSpacing(0.1), button_box, HSpacing(0.1)),
+        VSpacing(0.2)
+      )
+    end
+
+    def textmode
+      if Mode.commandline
```
Comment (lslezak) [inline library/packages/src/modules/PackageCallbacks.rb]: I think it could be simplified to
```ruby
Mode.commandline || UI.TextMode
```

Comment (jreidinger) [inline library/packages/src/modules/PackageCallbacks.rb]: good idea

**Comment by jreidinger** [comment]
@lslezak that is last minute fixes, will fix it. I am sure it will help when I write test if it works as expected

**Comment by lslezak** [comment]
LGTM, thanks!

**Pull request merged**